core/src/test/java/org/elasticsearch/cluster/ClusterChangedEventTests.java

```diff
@@ -54,11 +55,17 @@
 
     private static final ClusterName TEST_CLUSTER_NAME = new ClusterName("test");
     private static final String NODE_ID_PREFIX = "node_";
-    private static final String INITIAL_CLUSTER_ID = UUIDs.randomBase64UUID();
-    // the initial indices which every cluster state test starts out with
-    private static final List<Index> initialIndices = Arrays.asList(new Index("idx1", UUIDs.randomBase64UUID()),
-                                                                    new Index("idx2", UUIDs.randomBase64UUID()),
-                                                                    new Index("idx3", UUIDs.randomBase64UUID()));
```

Comment:
Otherwise these fields will be initialized in the class initializer which will not run under the randomized context and will lead to a security violation.